cli/tests/error_local_static_import_from_remote.js.out

```diff
@@ -0,0 +1,2 @@
+[WILDCARD]
+Remote module are not allowed to statically import local modules. Use dynamic import instead.
```

Comment:
In case of `.js`-only imports the error is very sparse and has no stack trace...

Reply:
Better than nothing